<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainshop</class>
 <widget class="QMainWindow" name="mainshop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1017</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1901</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="Logos">
         <property name="minimumSize">
          <size>
           <width>52</width>
           <height>52</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>52</width>
           <height>52</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>24</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Oriventa Shop</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="search">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>IBeamCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">border-top-left-radius: 10px;
border-bottom-left-radius: 10px;
border-right: none;
border: 1px solid black;
border-right: none;</string>
             </property>
             <property name="placeholderText">
              <string>Я шукаю...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Search_Button">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">#Search_Button {
border-top-right-radius: 10px;
border-bottom-right-radius: 10px;
border: 1px solid black;
border-left: none;
background-color: rgb(237, 203, 254);
}

#Search_Button:hover {
    background-color: rgb(0, 238, 255);
}

</string>
             </property>
             <property name="text">
              <string>Знайти</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="Button_user">
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>52</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#Button_user{
background-color: rgb(255, 255, 255);
border: none;
border-radius: 5px;
padding-left: 5px;
padding-right: 5px;
}

#Button_user:hover {
    background-color:rgb(0, 238, 255);
}

#Button_user:pressed {
    background-color:rgb(255, 255, 255);
}

QPushButton#Button_user::menu-indicator {
    image: none;
	width: 0;
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cart">
       <property name="maximumSize">
        <size>
         <width>52</width>
         <height>52</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#cart {
    background-color:rgb(255, 255, 255);
    color: white;
    font-size: 16px;
    padding: 10px;
    border-radius: 5px;
	border: none;
}

#cart:hover {
    background-color:rgb(0, 238, 255);
}

#cart:pressed {
    background-color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>1621</width>
      <height>871</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
border: none;
border-radius:10px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1621</width>
       <height>871</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>260</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>24</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-top-left-radius: 10px;
border-top-right-radius: 10px;
background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Каталог</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="categoryList">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor" stdset="0">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="focusPolicy">
          <enum>Qt::FocusPolicy::NoFocus</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QListWidget {
    border: none; 
    border-bottom-left-radius: 10px;
	border-bottom-right-radius: 10px;
    background-color: rgb(255, 255, 255); 
}

QListWidget::item {
    font-family: Arial;
    font-size: 12pt;
    text-align: center;
    background: transparent;
    background-color:rgb(87, 0, 134);
    color: white;
	padding: 5px;
	border-radius: 10px
}

QListWidget::item:selected {
   background-color:rgb(0, 238, 255);
    color: white;
}

</string>
         </property>
         <property name="movement">
          <enum>QListView::Movement::Static</enum>
         </property>
         <item>
          <property name="text">
           <string>Головна сторінка</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="background">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Електроніка</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Одяг</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
            <underline>false</underline>
            <strikeout>false</strikeout>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="foreground">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Товари для дому</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Побутова техніка</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Зоотовари</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Продукти</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Сантехніка</string>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>24</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Сортування</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sort1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#sort1 {
    background-color:rgb(237, 203, 254);
    padding: 10px;
    border-radius: 5px;
	border: none;
}

#sort1:hover {
    background-color:rgb(0, 238, 255);
}

#sort1:pressed {
    background-color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Від дорогих до дешевих</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sort2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#sort2 {
    background-color:rgb(237, 203, 254);
    padding: 10px;
    border-radius: 5px;
	border: none;
}

#sort2:hover {
    background-color:rgb(0, 238, 255);
}

#sort2:pressed {
    background-color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Від дешевих до дорогих</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#resetButton {
    background-color:rgb(237, 203, 254);
    padding: 10px;
    border-radius: 5px;
	border: none;
}

#resetButton:hover {
    background-color:rgb(0, 238, 255);
}

#resetButton:pressed {
    background-color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Скинути</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>957</y>
      <width>1920</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Контактна інформація: | oriventashop@gmail.com | https://github.com/Danil363/Project3 |</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
